lib/index.ts

```diff
@@ -31,44 +32,67 @@ async function getBrowser(
     env = "LOCAL";
   }
 
-  if (env === "BROWSERBASE" && !process.env.BROWSERBASE_PROJECT_ID) {
```

Comment:
I'm assuming this was removed bc you don't need project ID to connect to a running session, but I don't see this checking for project ID before creating session (see my other comment)